src/settings.py

```diff
@@ -188,6 +186,16 @@ def get_arg_specific_settings(args: Dict[str, Any]) -> Dict[str, Any]:
         """A workaround for global settings that are derived from the
         experiment-specific args."""
         return dict(
+            # For learning-based approaches, the data collection timeout for
+            # planning.
+            offline_data_planning_timeout=defaultdict(
```

Comment:
why can't we just make this default across the board? none of our stuff really cares about if training might take a little bit longer right, and usually it'll succeed very fast anyway